# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1294992409 18000
#      Fri Jan 14 03:06:49 2011 -0500
# Node ID 817f27bf0216e95508c6b1ac67ad5f72c4bcb210
# Parent  00fe5069b70e8365fb7b623f79a727988ebbf501
fix setting of gnulib default directory in bootstrap script

diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,8 +1,13 @@
+2011-01-14  John W. Eaton  <jwe@octave.org>
+
+	* bootstrap (gnulib_path): Set default gnulib path with ${x:=y},
+	not ${x=y}.
+
 2011-01-14  John W. Eaton  <jwe@octave.org>
 
 	* bootstrap, bootstrap.conf: Update from gnulib sources.
 
 2011-01-13  John W. Eaton  <jwe@octave.org>
 
 	* bootstrap.conf (gnulib_modules): Include mkstemp in the list.
 
diff --git a/bootstrap b/bootstrap
--- a/bootstrap
+++ b/bootstrap
@@ -457,17 +457,17 @@ cleanup_gnulib() {
   exit $status
 }
 
 git_modules_config () {
   test -f .gitmodules && git config --file .gitmodules "$@"
 }
 
 gnulib_path=`git_modules_config submodule.gnulib.path`
-: ${gnulib_path=gnulib}
+: ${gnulib_path:=gnulib}
 
 # Get gnulib files.
 
 case ${GNULIB_SRCDIR--} in
 -)
   if git_modules_config submodule.gnulib.url >/dev/null; then
     echo "$0: getting gnulib files..."
     git submodule init || exit $?
